distribution/pom.xml

```diff
@@ -258,6 +258,8 @@
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions:druid-kubernetes-extensions</argument>
                                         <argument>-c</argument>
+                                        <argument>org.apache.druid.extensions:druid-iceberg-extensions</argument>
```

Comment:
this is a contrib extension so we shouldn't be shipping it in the distribution bundle.